src/core/server/saved_objects/migrations/core/build_active_mappings.ts

```diff
@@ -164,7 +164,7 @@ function defaultMapping(): IndexMapping {
           },
         },
       },
-      referencesMigrationVersion: {
+      coreMigrationVersion: {
         type: 'keyword',
       },
```

Comment:
In https://github.com/elastic/kibana/issues/77961#issuecomment-746566179 we are planning to add 'core' migrations to add a tiebreaker field to all SOs. Given the new name in current PR, would we be using the same mechanism then? cc @rudolf

Reply:
A while back, Rudolf and I spoke about this field being reused for other things such as the comment that you linked. I don't see why we couldn't use it for that!

Would we need to expose it to consumers in that case? If so, I'll need to rework some of my assumptions (RE: https://github.com/elastic/kibana/pull/80945#discussion_r537121688 I assumed this would be an implementation detail)

I still feel that "conversion transforms" and "reference transforms" should be hidden from consumers / only triggered when Kibana is migrating the entire index. However, we can add a fourth type of transform ("core transform"?) that is applied upon object creation. LMK what you think.

Reply:
> Would we need to expose it to consumers in that case?

For our tie breaker needs, we will, as this should be included in the export/import process.

> I still feel that "conversion transforms" and "reference transforms" should be hidden from consumers / only triggered when Kibana is migrating the entire index. However, we can add a fourth type of transform

I agree that what you did in the current PR should probably be dissociated from what we are planning to do with `core` migration, as these answers distinct use cases and are different requirements. So, the only thing bothering me there is the naming used here. `coreMigrationVersion` seems like a perfect name for what we will name 'core' migrations 😅 .